### ISSUE TITLE
A small typo

### ISSUE THREAD

**Issue by junming259** [vanhuyz/CycleGAN-TensorFlow#28]
Thanks for sharing your code, excellent job! I found a small typo in your code.

[https://github.com/vanhuyz/CycleGAN-TensorFlow/blob/master/ops.py#L84](url)

conv2 = tf.nn.conv2d(padded2, weights1, strides=[1, 1, 1, 1], padding='VALID')

It should be weights2 instead of weights1, right? 

Thanks

**Comment by vanhuyz** [comment]
Yes, `weights2` for sure. Please send me a pull request!

**Comment by vanhuyz** [comment]
Solved with #29.
